tests/test_barostat_water_density.py

```diff
@@ -36,17 +35,13 @@
 
     # generate an alchemical system of a waterbox + alchemical ligand:
     # effectively discard ligands by running in AbsoluteFreeEnergy mode at lambda = 1.0
-    mol_a, _, core, ff = hif2a_ligand_pair.mol_a, hif2a_ligand_pair.mol_b, hif2a_ligand_pair.core, hif2a_ligand_pair.ff
-    complex_system, complex_coords, complex_box, complex_top = build_water_system(
-        initial_waterbox_width.value_in_unit(unit.nanometer)
-    )
-
-    min_complex_coords = minimize_host_4d([mol_a], complex_system, complex_coords, ff, complex_box)
-    afe = AbsoluteFreeEnergy(mol_a, ff)
-
-    unbound_potentials, sys_params, masses, coords = afe.prepare_host_edge(
-        ff.get_ordered_params(), complex_system, min_complex_coords
+    mol_a, ff = (
+        hif2a_ligand_pair.mol_a,
+        hif2a_ligand_pair.ff,
     )
+    unbound_potentials, sys_params, masses, coords, complex_box = get_solvent_phase_system(mol_a, ff, margin=0.0)
```

Comment:
Low-priority question: Non-default `margin=0.0` significant?

Reply:
It's just to make the test return the same exact value as before my changes.

Reply:
Thanks -- makes sense!